libs/colbert/ragstack_colbert/base_retriever.py

```diff
@@ -27,7 +27,7 @@ def retrieve(
         self,
         query: str,
         k: Optional[int] = None,
-        query_maxlen: Optional[int] = None,
+        query_maxlen: int = 64,
```

Comment:
i think we have proven that dynamic query_maxlen is superior to using the default of 64. I'd undo this change.